docs/source/installation/build-from-source-linux.md

```
@@ -179,15 +180,15 @@ relevant classes. The associated unit tests should also be consulted for underst
 
 This feature will not be enabled when [`building only the C++ runtime`](#link-with-the-tensorrt-llm-c++-runtime).
```

markdownlint-cli2 (0.17.2)

Link fragments should be valid

(MD051, link-fragments)